### PR TITLE
[ADD] project_task_digitized_signature

### PR DESCRIPTION
**Pull request by cubells**
[WIP] Depends on
- [x] https://github.com/OCA/web/pull/666  

New module that allows to add digitized signature to tasks and send by mail.

cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
Add installation section in README indicating that it depends on `web_widget_digitized_signature` that is on OCA/web

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
There's already a task report on https://github.com/OCA/project-reporting/blob/9.0/project_task_report, so don't create another one.

File: project_task_digitized_signature/README.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+================================
+Project Task Digitized Signature
+================================
+
+This module adds a signature field to task report.
+
+Installation
+============
+
+This module depends on :
```
Comment (pedrobaeza) [inline project_task_digitized_signature/README.rst]: Add project_task_report dependency

File: project_task_digitized_signature/README.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+* web_widget_digitized_signature
+
+This module is part of the OCA/web suite.
+
+Configuration
+=============
+
+#. Go to **Settings > Users > Users** and set a digital signature for a user.
```
Comment (pedrobaeza) [inline project_task_digitized_signature/README.rst]: Update these instructions

File: project_task_digitized_signature/README.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/project-service/issues>`_. In case of trouble, please
```
Comment (pedrobaeza) [inline project_task_digitized_signature/README.rst]: s/project-service/project

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: project_task_digitized_signature/README.rst (hunk 1 above)
Comment (cubells) [inline project_task_digitized_signature/README.rst]: done.

File: project_task_digitized_signature/README.rst [hunk 4]
```diff
@@ -17,10 +17,13 @@ This module depends on :
...
 
+* project_task_report
+
+This module is part of the OCA/project-reporting suite.
+
 Configuration
 =============
 
-#. Go to **Settings > Users > Users** and set a digital signature for a user.
 #. Go to **Project > Search > Tasks** and print a task order. Signature will be
```
Comment (pedrobaeza) [inline project_task_digitized_signature/README.rst]: These instructions are still incomplete. Please put where to make the sign.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@chienandalu @rafaelbn please review

**Review by chienandalu — CHANGES_REQUESTED** [review]
A field name indicating what the field is for would be useful:
image
Couldn't test report printing due to error with wkhtmltopdf.
image

File: project_task_digitized_signature/report/project_task_report.xml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" ?>
+<odoo>
+
+    <template id="report_taskorder_document" inherit_id="project_task_report.task_report" priority="9999">
```
Comment (pedrobaeza) [inline project_task_digitized_signature/report/project_task_report.xml]: Don't put priority. Use it only when making `replace` operations to make sure they are performed after the rest of inheritance.

**Comment by pedrobaeza** [comment]
Agh, runbot still continues refusing to use correctly wkhtmltopdf. Any way, @chienandalu, I assure you it works :wink:

**Review by chienandalu — APPROVED** [review]
@pedrobaeza Ok, I take your word :smile:

**Pull request merged**